Marlin/ubl_motion.cpp

```diff
@@ -250,9 +231,8 @@
     const float m = dy / dx,
                 c = start[Y_AXIS] - m * start[X_AXIS];
 
-    const bool inf_normalized_flag = isinf(e_normalized_dist),
-               inf_m_flag = isinf(m);
-
+    const bool inf_normalized_flag=isinf(e_normalized_dist),
+               inf_m_flag=isinf(m);
```

Comment:
If the pre-initialized variable declarations are being put on separate lines...   They look like assignments.   It just makes sense to have both consistent.